scripts/jenkins-yoctobuild-build.sh

```diff
@@ -2,19 +2,24 @@
 
 set -e -x -E
 
-$WORKSPACE/mender-qa/scripts/wait-until-build-host-ready.sh
+if [ -n "$JENKINS_URL" ]; then
+    $WORKSPACE/mender-qa/scripts/wait-until-build-host-ready.sh
+fi
 
 echo $WORKSPACE
 
-sudo apt-get -qy --force-yes install docker-ce || apt-get -qy --force-yes install docker-ce
-# make sure Docker uses the block storage to store docker containers.
-sudo bash -c 'cat << EOF > /etc/docker/daemon.json
-{
-"data-root": "/var/lib/docker/",
-"storage-driver": "overlay2"
-}
-EOF'
-sudo service docker restart
+# Jenkins builds will install docker here. Gitlab runners have it already in the image
+if [ -n "$JENKINS_URL" ]; then
+    sudo apt-get -qy --force-yes install docker-ce || apt-get -qy --force-yes install docker-ce
+    # make sure Docker uses the block storage to store docker containers.
+    sudo bash -c 'cat << EOF > /etc/docker/daemon.json
+    {
+    "data-root": "/var/lib/docker/",
+    "storage-driver": "overlay2"
+    }
+    EOF'
```

Comment:
By the way, this wrong... `bash: line 5: warning: here-document at line 0 delimited by end-of-file (wanted 'EOF')`

Do you agree that that might very well be the reason why so many integ tests fail and the builds just timeouts after 480 minutes?

Reply:
I fixed it on commit a96c6ff

Reply:
It might be, I think it probably left a literal `EOF` in the file. Definitely the new, non-indented version is the correct one.